test/intl402/Temporal/ZonedDateTime/compare/argument-propertybag-timezone-string-datetime.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.zoneddatetime.compare
+description: Conversion of ISO date-time strings to Temporal.TimeZone instances (with Intl time zones)
```

Comment:
Minor nit: Intl is not required to use IANA time zones, so maybe this title (and others like it, if any) should say "IANA" instead of "Intl" ?

Reply:
I'll tack on another commit that replaces "Intl time zone" with "IANA time zone" everywhere.